### PR TITLE
Use common awslogs role for cloudwatch log pushing, conditionally

### PR DESCRIPTION
**Pull request by vlfig**
Depend on it conditionally.
Setup apache-specific logging configuration conditionally.
Dependents have to opt in. No change of behaviour if they don't change.

### CODE REVIEW

File: tasks/logging.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# awslogs_agent_config_dir variable comes form awslogs role
+- name: "Install the apache config template into {{ awslogs_agent_config_dir }}"
+  copy:
+    src: apache-awslogs-agent.conf.j2
+    dest: "{{ awslogs_agent_config_dir }}/apache.conf.j2"
+
+- name: Add log_group configuration block for apache
+  blockinfile:
+    dest: "{{ scripts_dir }}/logs_group_configuration.py"
```
Comment (codesplicer) [inline tasks/logging.yml]: Where is `scripts_dir` being set? Not in the defaults file

Comment (vlfig) [inline tasks/logging.yml]: in the awslogs role

Comment (codesplicer) [inline tasks/logging.yml]: In general I try to create a subdirectory under `{{ scripts_dir }}` for the role script\s to live in.

**Comment by vlfig** [comment]
@codesplicer https://github.com/webhop/ansible-awslogs/commit/b7df22a4429628c5f86bb1234c5e9212a5078281 makes awslogs_scripts_dir point to a role-specific folder

**Pull request merged**